<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoubleGameChoose</class>
 <widget class="QWidget" name="DoubleGameChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_port">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ip">
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_ip"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_port"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>opponent </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_other"/>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>isReady</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="text">
       <string>刷新在线人员</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_connect">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_quitconnect">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_messageInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit_status">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>40</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>259</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>选择你的棋子颜色</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>19</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_black">
       <property name="text">
        <string>黑色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_white">
       <property name="text">
        <string>白色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_random">
       <property name="text">
        <string>随机</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_isForbidden">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>是否禁手</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
